pylua/test/websocket_test.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+import thread
+import time
+import lua
+from pylua.load_lua import open_script
+from pylua.network import * #pylint: disable=wildcard-import, unused-wildcard-import
+
+class TestWebsocket(object):
+    def test1(self): #pylint: disable=no-self-use
+        server = LuaWebSocketServer()
+        thread.start_new_thread(server.run_server, ())
+        time.sleep(2)
+        web_socket_client = LuaWebSocketClient()
+        web_socket_client.open("ws://localhost:8087/")
+        web_socket_client.write("test2")
+        print web_socket_client.read()
+        web_socket_client.close()
+
+    def test2(self): #pylint: disable=no-self-use
+        script = open_script('pylua/test/websocket_test1.lua')
+        lua.execute(script) #pylint: disable=no-member
```

Comment:
@OHerasym, what is the meaning of warning you've suppressing? I think idea to suppress all encountered warnings is not so good. Better to fix them.

How lua will fail the test case?